lib/calc.rb

```diff
@@ -1,3 +1,38 @@
 # frozen_string_literal: true
 
 # Calculator Class here
+class Calculator
+  attr_accessor addition
```

Comment:
These should be methods not attribute accessors.  They should be named `add`, `subtract`, `multiply`, and `divide`.